cafebabel/static/js/edit-article.js

```diff
@@ -27,68 +39,108 @@ class Tags {
   _createTag(tagValue, index) {
     const li = document.createElement('li')
     const input = document.createElement('input')
+    const a = document.createElement('a')
     input.type = 'hidden'
     input.name = `tag-${index + 1}`
     input.setAttribute('list', 'tags')
     input.value = tagValue
     li.append(tagValue)
     li.append(input)
+    li.append(a)
```

Comment:
`a` seems like blank with no attribute. Is it normal?

We could also consider a JSX-style writing which might be more readable:
```js
return `
  <li>
    <input type=hidden list=tags name=tag-${index + 1}>
    <a></a>
  </li>
`
```

Reply:
We don't find any solution to attach event on the pseudo-element ::after.
After tries we add an `a` to attach a -remove link-